examples/upgrade_forseti_with_v5.0/tutorial.md

```diff
@@ -0,0 +1,210 @@
+strategy called
+[Application Default Credentials](https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application)
+when provisioning resources. To enable this approach, set the
+appropriate environment variable to the path of the service account JSON
+key file:
+
+```sh
+export GOOGLE_APPLICATION_CREDENTIALS="PATH_TO_JSON_KEY_FILE"
+```
+## Backup the Terraform State
+In a shell, navigate to the folder containing your user-defined Terraform module, most likely in a **main.tf**.
+
+The following command will simultaneously backup your existing Terraform state and remove resource from the current state file.  This will not affect your existing Forseti deployment.
+```sh
+terraform state rm $(terraform state list)
+```
+## Update main.tf
+In order to support this upgrade, we'll need to update a few input variables.
+
+### Source
+The **source** will need to point to the root terraform-google-forseti module.
+```
+source = "terraform-google-modules/forseti/google"
+```
+If you have cloned the module to your local file system, you may set the **source** path to 
+the directory containing the module.
+
+### Version
+The **version** will need to be 5.0.0.
+```
+version = "5.0.0"
+```
+If you have set the **source** path to the directory containing the module, omit the **version** variable.
+
+### Region
+If you have a **region**, it will need to be split into the cloudsql_region, server_region, and client_region variables.
+
+Before (example):
+```
+region = "us-central1"
+```
+After (example):
+```
+cloudsql_region = "us-central1"
+server_region   = "us-central1"
+client_region   = "us-central1"
+```
+### Credentials path
+Remove the **credentials_path** variable if present.  The `google` provider now solely relies on the _GOOGLE_APPLICACTION_CREDENTIALS_ environment variable.
+
+### Root Resource identity
+Add the ***resource_name_suffix** variable and set it to the resource suffix.  The suffix can be found appended to the Forseti Server VM, for example.
+```
+resource_name_suffix = "abc123efg" 
+```
+## Obtain and Run the Import Script
+### Obtain the Import Script
+This [import script](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/import.sh) will import the Forseti GCP resources into a local state file.
+
+```sh
+curl --location --remote-name https://raw.githubusercontent.com/forseti-security/terraform-google-forseti/master/helpers/import.sh
+chmod +x import.sh
+./import.sh -h
+```
+### Initialize the Terraform Module
+```sh
+terraform init
+```
+
+### Import the Existing Terraform State
+Import the existing resources to the Terraform state, replacing the
+uppercase values with the aforementioned values:
+
+```sh
+./import.sh -m MODULE_LOCAL_NAME -o ORG_ID -p PROJECT_ID -s RESOURCE_NAME_SUFFIX -z GCE_ZONE [-n NETWORK_PROJECT_ID]
+```
+
+Observe the expected Terraform changes by execution `terraform plan`.  As stated in the introduction, if you have any
```

Comment:
A quick blurb about what people should observe?

Reply:
This text is misplaced and duplicated.  Removing.